Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -246,9 +249,10 @@ private void ComputeControlGroupingIfNecessary()
 
```

Comment:
Wondering if this Resize could be moved out of the nested for loop, e.g. by. spinning round in advance to calculate max size.

Reply:
Also wondering if the actionCountInGroup index overflow the tempActionGroupIndexes above.
Perhaps splitting the loop could allow a sanity check (or perhaps a sanity check could be added without splitting the loop)